.gitignore

```diff
@@ -1 +1,2 @@
 /vendor
+composer.lock
```

Comment:
Please remove, without the lock present the package manager may install incompatible dependency versions.

Reply:
No problem